java-strings/src/main/java/com/baeldung/stringduplicates/RemoveDuplicateFromString.java

```diff
@@ -0,0 +1,102 @@
+package com.baeldung.stringduplicates;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class RemoveDuplicateFromString {
+
+
+    String removeDuplicatesUsingCharArray(String str) {
+
+        char[] chars = str.toCharArray();
+        StringBuilder sb = new StringBuilder();
+        int repeatedCtr;
+        for (int i = 0; i < chars.length; i++) {
+            repeatedCtr = 0;
+            for (int j = i + 1; j < chars.length; j++) {
+                if (chars[i] == chars[j]) {
+                    repeatedCtr++;
+                }
+            }
+            if (repeatedCtr == 0) {
+                sb.append(chars[i]);
+            }
+        }
+        return sb.toString();
+    }
+
+    String removeDuplicatesUsinglinkedHashSet(String str) {
+
+        StringBuilder sb = new StringBuilder();
+        Set<Character> linkedHashSet = new LinkedHashSet<>();
+
+        for (int i = 0; i < str.length(); i++) {
+            linkedHashSet.add(str.charAt(i));
+        }
+
+        for (Character c : linkedHashSet) {
+            sb.append(c);
+        }
+
+        return sb.toString();
+    }
+
+    String removeDuplicatesUsingSorting(String str) {
```

Comment:
As @jzheaux mentioned this method will still throw an exception if passed an empty string. Can you fix this up?

Reply:
will add an if statement

Reply:
Done